authors/apps/articles/views.py

```diff
@@ -129,3 +133,20 @@ def destroy(self, request, *args, **kwargs):
         super().destroy(self, request, *args, **kwargs)
 
         return Response({'message': 'The article has been deleted.'})
+
+
+class SearchFilterListAPIView(ListAPIView):
+    """
+    Performs search and filter of articles using parameters
+    """
+    serializer_class = ArticleSerializer
+    permission_classes = (AllowAny,)
+    queryset = Article.objects.all()
+
+    filter_backends = (DjangoFilterBackend, SearchFilter, OrderingFilter)
+    # filter fields are used to filter the articles using the tags, author's username and title
+    filter_fields = ('tags__tag', 'author__username', 'title')
```

Comment:
let's use `tag, author_username` and `title` as the fields instead.

Reply:
We used tags__tag because the metafields contains fields that are not defined on this FilterSet: tag...unless you have a workaround to this